elm-app/src/elm/Event/Test.elm

```diff
@@ -60,7 +60,25 @@ viewEventTest =
                 viewEvent baseUrl English event1
                     |> Query.fromHtml
                     |> Query.find [ Selector.class "event-date" ]
-                    |> Query.has [ text "" ]
+                    |> Query.children [ tag "span" ]
+                    |> Query.first
+                    |> Query.has [ text "Thu, 01/01/1970 02:00" ]
+        , test "Event with the same Date and different Time" <|
```

Comment:
Might need a different [package](http://package.elm-lang.org/packages/rluiten/elm-date-extra/9.0.0) for this. But that's too complicated.

So instead, we could check in the test the date doesn't appear twice. If you are able to achieve in timebox, great -- otherwise, please open a follow up issue for this, and remove test from here for now.